### DISCUSSION TITLE
New Fun Mod For Lazer

### DISCUSSION THREAD

**Discussion by Axeanz** [ppy/osu#13060] [Ideas]
A mod called "Freeze".

Maybe a weird idea, to make your screen(or just cursor) freeze for a bit, but it kind of works. 
I got this idea, mainly because when I play osu!Lazer, my screen freezes pretty easily, but somehow it was still sort of playable, depending on how long it froze. As long as it's playable, and not too annoying, it may work right? I think freezing the whole screen might be too much or too annoying for an whole map, so just the cursor freezing might be an option. And then we have a mod that changes something with the cursor.

("cursor" can be replaced with screen)
The bpm and/or length+difficulty can decide how often the cursor will freeze. Higher bpm and/or shorter maps with high difficulty= more frequent freezes.
The AR can decide how long the cursor will freeze. AR 7 freezes the cursor longer than AR 10.
The OD can decide how much 'recovery time' you get. Higher OD= less recovery time. (kind of the same as more frequent freezes)

If just the cursor will freeze:
- The approach-circles will still move into the circle.
- New circles will still appear with moving approach-circles.
- Cursor isn't ''seen'' moving, but will be put back to it's actual place after unfreezing.
- A secundairy option can be that instead of freezing, the cursor will dissapear, but may cause problems because cursortrails.

If the screen will freeze:
- New circles will not appear until unfrozen, meaning the screen may not freeze too long.
- The approach-circles will not move into the circle.
- The cursor may or may not move, depending on how difficult it is with the cursor freezing.

For both:
- The circles will dissapear after being hit or missed.
- There will not be a forewarning for a freeze, making it unexpected.
- The HUD will still update through the freeze.
- The screen/cursor could have a frozen look or some effects.
- Like all other Fun mods, it will be unranked without a score boost.

I don't know if it would be a mod that is playable, but the new 'blinds' mod is also kind of impossible, so I can always give it a shot to request. 
If it's totally impossible to do because of game-coding or how games work; I have no experience with coding or logics behind the games.  I will try to update if there are incorrect things about this, or if there are better ways to do things.

I hope it will be considered.

Notice me Peppy-senpai <33

**Comment by frenzibyte** [comment]
Interesting concept. would like to add effects to the screen as the cursor freezes and potentially to the track. :) (for making the mod more fun than simulating lag)

**Comment by mouzedrift** [comment]
i was instantly thinking about a lag that happened to me once https://clips.twitch.tv/RichAntsyElephantNinjaGrumpy (skip to 5 sec) kinda weird but that could be fun

**Comment by Axeanz** [comment]
> 
> 
> i was instantly thinking about a lag that happened to me once https://clips.twitch.tv/RichAntsyElephantNinjaGrumpy (skip to 5 sec) kinda weird but that could be fun

Something like that, but that length would be extreme to do for the entire screen. (unless it's going to be a mod like blinds)

**Comment by HoutarouOreki** [comment]
How about making the cursor update its position only on click? So you click to hit a circle, and you can see whether you hit it or how much off you were. Otherwise the cursor doesn't move at all.

**Comment by Axeanz** [comment]
> 
> 
> How about making the cursor update its position only on click? So you click to hit a circle, and you can see whether you hit it or how much off you were.

As in it updates with EVERY click (which is easy to abuse) or only on a circlehit? 
I think it still depends on how long the freezing will be, if its for longer than 5 seconds or something, it should definitly be a thing on a circlehit, but otherwise I think it can go fine without that.

**Comment by aqilc** [comment]
~~I like how you guys are not trying to fix glitches, but add them as a feature~~
